tests/api/test_security.py

```diff
@@ -13,6 +13,7 @@ def test_main_page(self, session, flask_api_url):
         assert response.status_code == 200
 
     def test_login_fake_user(self,session, flask_api_url):
+        session = requests.Session()
```

Comment:
I added it to every method and added login after, / one time tried to reinitialize session variable, but it doesn't work anyway